### ISSUE TITLE
Change 'Change' button text to 'View' on admin index page if user only has view perm for model

### ISSUE THREAD

**Issue by ciarancourtney** [ctxis/django-admin-view-permission#28]
Somewhat related to https://github.com/ctxis/django-admin-view-permission/issues/5

**Comment by lefterisnik** [comment]
Yes, I know. This is annoying. I need to override the django templates. I will give it a try.

**Comment by ciarancourtney** [comment]
@lefterisnik I did try change the django template, although its not an officially 'override-able' template, so it would be difficult to maintain between versions
